_includes/common/ui/statements-page.md

```diff
@@ -10,9 +10,17 @@
 
 By default, the Statements page shows SQL statements from all applications and databases running on the cluster.
 
-You can search for statements using the date field or the search field.
+### Search field
 
-To search by date, pick a date range that is within the time period since the statistics were last cleared. Click **reset time** to reset the date.
+To search using the search field, type a string over `Search Statements` and press `Enter`. The list of statements is filtered by the string.
+
+### Date range
+
+To search by date, click the date range selector and pick a date range that is within the time period since the statistics were last cleared. Click **reset time** to reset the date to the last hour.
```

Comment:
Should we remove "since the statistics were last cleared" ? It can be since the last time rows were cleared _or_ never cleared but we store up to one million rows before entries start getting purged. Might be confusing if the user never cleared the statistics.